paderbox/transform/module_fbank.py

```diff
@@ -73,7 +88,30 @@ def ifbanks(self):
         return np.linalg.pinv(self.fbanks.T).T
 
     def __call__(self, x):
-        x = np.dot(x, self.fbanks)
+        if self.warping_fn is None:
+            x = x @ self.fbanks
+        else:
+            independent_axis = [ax if ax >= 0 else x.ndim+ax for ax in self.independent_axis]
```

Comment:
Is `[ax % x.ndim for ax in self.independent_axis]` more obvious?

Reply:
What if `ax < -x.ndim`?